libraries/AdaptiveExpressions/Expression.cs

```diff
@@ -530,6 +530,18 @@ public void ValidateTree()
         public (T value, string error) TryEvaluate<T>(IMemory state, Options options = null)
         {
             var opts = options ?? new Options();
+            if (opts.Locale != null)
+            {
+                try
```

Comment:
simplify the empty try catch to -> "new CultureInfo(opts.Locale);"

Reply:
fixed